scripts/evaluation.fig12.C

```diff
@@ -68,14 +68,14 @@ void evaluation(const char *ifname, const char *ofname = 0)
 
 #ifdef _USE_RECONSTRUCTED_TRACKS_
     // First populate the reconstructed-to-simulated particle mapping table;
-    std::map<eic::Index, const eic::ReconstructedParticleData*> mc2rc;
+    std::map<eic::Index, const edm4eic::ReconstructedParticleData*> mc2rc;
     for(const auto &rctrack: *rctracks) 
       mc2rc[rctrack.mcID] = &rctrack;
 #endif
     
     // Then the Cherenkov-to-reconstructed mapping; FIXME: may want to use Cherenkov-to-simulated 
     // mapping to start with, for the debugging purposes;
-    std::map<eic::Index, const eic::CherenkovParticleIDData*> rc2cherenkov;
+    std::map<eic::Index, const edm4eic::CherenkovParticleIDData*> rc2cherenkov;
     for(const auto &pid: *cherenkov) 
       rc2cherenkov[pid.recID] = &pid;
```

Comment:
this script is also likely broken, and will need the same refactoring as `evaluation.C` and `reader.C` needed